### PR TITLE
[WC-109] Add documentation for refactored progress bar widget

### PR DESCRIPTION
**Pull request by Keraito**
Similar to https://github.com/mendix/docs/pull/3086:

The progress bar widget has been converted to a pluggable widget that will only be supported on MX 9 and above. I kept the old widget documentation as well, since mx developers can still use these widgets with mx 7 or 8.

### CODE REVIEW

File: content/appstore/widgets/progress-bar.md [hunk 1]
```diff
@@ -10,17 +10,53 @@ tags: ["marketplace", "app store", "marketplace component", "app store component
...
+
+1. Whether to show a label.
+1. The type of the label:
+   1. Text: A custom text.
+   1. Percentage: The **Current value** converted onto the configured range as a percentage. 
+   1. Custom: A custom widget.
+
+### 2.2 Styling
+
+Under the **Appearance** tab, the following styling properties can be adjusted to the progress bar:
```
Comment (jordanch) [inline content/appstore/widgets/progress-bar.md:35]: ..."adjusted **for** the progress bar" is maybe better?

**Review by jordanch — CHANGES_REQUESTED** [review]
minor grammatical change, elsewise LGTM 👍

File: content/appstore/widgets/progress-bar.md (hunk 1 above)
Comment (Keraito) [reply content/appstore/widgets/progress-bar.md]: I got stuck in my head with this verb in the context of "_adjust to_ the situation". Changed it! I also assume someone will go over this language/grammar wise cuz of the "Needs language review" label 😄

Comment (ConnorLand) [reply content/appstore/widgets/progress-bar.md]: "Adjusted for" the progress bar is indeed better. Verb + preposition idioms in English are so tricky. Thanks for the good PR.

**Review by ConnorLand — APPROVED** [review]

**Comment by Keraito** [comment]
@jordanch I updated this based on the new design specs per https://github.com/mendix/widgets-resources/pull/496. Could you re-review? The commit that matters is 9ec464f.

**Review by ConnorLand — APPROVED** [review]

**Review by jordanch — APPROVED** [review]

**Pull request merged**